html_challange1.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Reminder: Properly indent your code, just like in javascript if you have HTML tags inside other tags indent them.